db/models.py

```diff
@@ -50,3 +53,65 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        to=settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        related_name="orders"
+    )
+
+    class Meta:
+        ordering = ("-created_at",)
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(
+        to=MovieSession, on_delete=models.CASCADE, related_name="tickets"
+    )
+    order = models.ForeignKey(
+        to=Order, on_delete=models.CASCADE, related_name="tickets"
+    )
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    def clean(self) -> None:
+        errors = {}
+
+        if isinstance((self.row, self.seat), int):
+            raise ValidationError("Row and seat numbers must be integers.")
```

Comment:
The condition `isinstance((self.row, self.seat), int)` is incorrect. It checks if a tuple is an instance of int, which will always be False. You should check each field individually, like `isinstance(self.row, int)` and `isinstance(self.seat, int)`.